# As talked with Matthew Barnes (from upstream) this way to set version causes 
# problems as results in a "3.1" slipping into the EDS_CHECK_VERSION preprocessor 
# macro, and preprocessor macros don't like floating point numbers, causing problems
# like mail-notification e-d-s patch not working when building against this e-d-s version.
# This patch appends .1 to the version number properly.
--- configure.ac~	2010-03-02 07:29:38.000000000 +0100
+++ configure.ac	2010-04-29 22:13:24.000000000 +0200
@@ -1,7 +1,7 @@
 dnl Evolution-Data-Server version
 m4_define([eds_major_version], [2])
 m4_define([eds_minor_version], [28])
-m4_define([eds_micro_version], [3.1])
+m4_define([eds_micro_version], [3])
 
 m4_define([eds_version],
 	[eds_major_version.eds_minor_version.eds_micro_version])
@@ -19,7 +19,7 @@
 
 dnl Autoconf / Automake Initialization
 AC_PREREQ(2.58)
-AC_INIT([evolution-data-server],[eds_version],[http://bugzilla.gnome.org/enter_bug.cgi?product=Evolution-Data-Server])
+AC_INIT([evolution-data-server],[eds_version].1,[http://bugzilla.gnome.org/enter_bug.cgi?product=Evolution-Data-Server])
 AM_INIT_AUTOMAKE([gnu 1.9])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR(README)
